src/Events/SEO/No_Index.php

```diff
@@ -0,0 +1,204 @@
+<?php
+/**
+ * Functionality for interacting with the SEO nodindex.
+ *
+ * @since   6.1.0
+ *
+ * @package TEC\Events\SEO
+ */
+
+namespace TEC\Events\SEO;
+
+use \Tribe__Date_Utils as Dates;
+
+ /**
+  * Class Provider
+  *
+  * @since   6.1.0
+  * @package TEC\Events\Telemetry
+  */
+class No_Index {
+	/**
+	 * Runs on the "wp" action. Inspects the main query object and if it relates to an events
+	 * query makes a decision to add a noindex meta tag based on whether events were returned
+	 * in the query results or not.
+	 *
+	 * @since ??
```

Comment:
This needs filling, if you cannot find the original implementation, then just drop this `@since` tag.